cdap-ui/app/features/flows/controllers/detail-ctrl.js

```diff
@@ -58,11 +59,13 @@ angular.module(PKG.name + '.feature.flows')
     });
 
     $scope.do = function(action) {
+      $scope.status = action;
```

Comment:
This is not something that we get from backend. I believe it was said that flows cannot have multiple runs. So I am listening for active runs from backend.

if active runs is 0 && status is null
  show start button
else if active runs is gt 0  && status is null
  show start button
if status is start
  show start loading 
if status is stop
  show stop loading